connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/VersionedPluginLoadingException.java

```diff
@@ -0,0 +1,32 @@
+package org.apache.kafka.connect.runtime.isolation;
+
+import org.apache.kafka.connect.errors.ConnectException;
+
+import java.util.List;
+
+public class VersionedPluginLoadingException extends ConnectException {
```

Comment:
Does this make more sense as a ConfigException?

Reply:
I'm okay with making it an extension of `ConfigException`. But let's keep it as a separate class type as it makes the errors more specific and helps in debugging. It also provides a set of available versions which is useful in other modules which capture this exception.

Reply:
Yeah I was imagining `extends ConfigException`.